infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -84,6 +84,11 @@ public void syncEvent(final UInt64 nodeSlot, final UInt64 headSlot, final int nu
     info(syncEventLog, Color.WHITE);
   }
 
+  public void syncCompleted() {
+    final String syncCompleteMsg = String.format("Syncing completed");
+    info(syncCompleteMsg, Color.GREEN);
```

Comment:
Actually one more nit on this - we don't need to use `String.format` because it's just a plain string.
```suggestion
    info("Syncing completed", Color.GREEN);
```